src/virtual/virtualAnalyticsData.ts

```diff
@@ -98,6 +99,44 @@ export async function fetchAnalyticsData(geb: Geb): Promise<AnalyticsData> {
             {}
         )
 
+    //TODO: Update liquidity pool addresses to reflect accurate contract addresses
+    const OD_ETH_LIQUIDITY_POOL = '0xEe01c0CD76354C383B8c7B4e65EA88D00B06f36f'
+    const ODG_ETH_LIQUIDITY_POOL = '0xEe01c0CD76354C383B8c7B4e65EA88D00B06f36f'
+
+    const tokenDetailsOD = getTokenDetails('arbitrum-goerli', 'OD');
+    const tokenDetailsODG = getTokenDetails('arbitrum-goerli', 'ODG');
+    const tokenDetailsWETH = getTokenDetails('arbitrum-goerli', 'WETH');
+
+    const tokenAddressOD = tokenDetailsOD.address;
+    const tokenAddressODG = tokenDetailsODG.address;
+    const tokenAddressWETH = tokenDetailsWETH.address;
+
+    const getBalanceOfTokenInLiquidityPool = async (tokenAddress: string, liquidityPoolAddress: string) => {
+        try {
+            const tokenContract = geb.getErc20Contract(tokenAddress)
+            return await tokenContract.balanceOf(liquidityPoolAddress);
+        } catch (e) {
+            console.log(e);
+            return null;
+        }
+    };
+
+    const ODBalance = await getBalanceOfTokenInLiquidityPool(tokenAddressOD, OD_ETH_LIQUIDITY_POOL);
+    const ODGBalance = await getBalanceOfTokenInLiquidityPool(tokenAddressODG, ODG_ETH_LIQUIDITY_POOL);
+    const wETHBalance_OD = await getBalanceOfTokenInLiquidityPool(tokenAddressWETH, OD_ETH_LIQUIDITY_POOL);
+
+    const ODMarketPrice = decoded?.tokenAnalyticsData['OD']?.currentPrice
+    const ODGMarketPrice = decoded?.tokenAnalyticsData['ODG']?.currentPrice
```

Comment:
There is no delayed oracle for OD and ODG, since they are not collateral for the protocol. Therefore there won't be any data returned in `tokenAnalyticsData`. 

However, this `fetchAnalyticsData` returns `marketPrice`, which is the market price for OD. 

For ODG, we will need to use the price returned from the camelot pool contract.